tests/sentry/options/test_store.py

```diff
@@ -17,9 +17,9 @@
 class OptionsStoreTest(TestCase):
     @fixture
     def store(self):
-        return OptionsStore(
-            cache=RedisCache()
-        )
+        c = LocMemCache('test', {})
```

Comment:
These are great because Django's `LocMemCache` actually shares a global key space outside of each instance. I don't really know why, but withotu clearing, values bleed over.